config/config.yaml

```
@@ -61,6 +61,7 @@ diffexp:
     volcano-plot: 0.05
     ma-plot: 0.05
     qq-plot: 0.05
+  pre-define-genelist: "resources/genelist.tsv"
```

```suggestion
  # Optional (comment in to use): provide a list of genes that shall be shown in a heatmap
  # and for which bootstrap plots (see below) shall be created.
  # pre-define-genelist: "resources/genelist.tsv"
```

And below, in the bootstrap_plots section, remove the genes_of_interest key and adapt the code.

---

And use it for both, the bootstrap plots and the heatmap.

---

This also has to be reflected in the schema.

---

Done.